README

```diff
@@ -24,25 +24,43 @@ For more information and details on how to contribute to `htop` visit [htop.dev]
 
 ## Build instructions
 
-This program is distributed as a standard GNU autotools-based package.
-
-Compiling `htop` requires the header files for `ncurses` (libncursesw*-dev). Install these and other required packages for C development from your package manager.
-
-Then, when compiling from a [release tarball](https://github.com/htop-dev/htop/releases), run:
-
+### Prerequisite
+List of build-time dependencies:
+*  `build-essential` standard GNU autotools-based
+*  `autoconf`
+*  `autotools`
+*  `ncurses` (libncursesw*-dev)
+
+**Note about `ncurses`:**
+> htop requires ncurses 6.0, the appropriate package is often enough still called libncurses5 (+-dev for headers). Also ncurses usually comes in two flavours:
+>* With Unicode support.
+>* Without Unicode support.
+>
+> This is also something that is usually reflected in the package name (the additional w).
```

Comment:
I think both the odd 5 vs 6 numbering and the 'w' package naming are Debian/Ubuntu-specific packaging quirks FWIW.  So perhaps reflect this here - instead of 'is usually reflected' say 'is reflected in [...] on Debian/Ubuntu.'